local-dev/templates/rawls.conf

```diff
@@ -1,34 +1,26 @@
-{{- $appsDomain := "test.firecloud.org" -}}
-{{- $appsSubDomain := printf "dev.%s" $appsDomain -}}
```

Comment:
These are all just to prevent typing the same thing over and over again - and presumably to make it easy to replace if needed for some reason - but I figured it wasn't necessary and just put the values in directly